Formula/wskdeploy.rb

```diff
@@ -1,9 +1,9 @@
 class Wskdeploy < Formula
   desc "Apache OpenWhisk project deployment utility"
   homepage "https://openwhisk.apache.org/"
-  url "https://github.com/apache/incubator-openwhisk-wskdeploy/archive/0.9.8-incubating.tar.gz"
-  version "0.9.8"
-  sha256 "10ff548bf43448d734e0e6f4b6aee8d306759a4ff8a5a7deaa61329c0f3376a6"
+  url "https://github.com/apache/incubator-openwhisk-wskdeploy/archive/0.9.9.tar.gz"
+  version "0.9.9"
```

Comment:
You can remove this `version ...` line now that the URL is one that Homebrew can parse automatically. This will fix the audit failure.